torchao/_models/mixtral-moe/generate.py

```diff
@@ -474,6 +477,12 @@ def callback(x):
         action="store_true",
         help="Whether to compile the prefill (improves prefill perf, but higher compile times)",
     )
+    parser.add_argument(
+        "--compile_mode",
+        type=str,
+        default="reduce-overhead",
+        help="which torch.compile mode to use: reduce-overhead or max-autotune, does nothing if --compile is not set.",
```

Comment:
do we expect a meaningful % of users to use max-autotune? if not, I'd say p1 and do it later.